core/settings.py

```diff
@@ -146,10 +146,19 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": "db.sqlite3",
+        "ENGINE": (
+            "django.db.backends.postgresql"
+            if os.environ.get("RDS_INSTANCE_ADDRESS")
+            else "django.db.backends.sqlite3"
+        ),
+        "NAME": os.environ.get("DATABASE_NAME", BASE_DIR / "db.sqlite3"),
+        "USER": os.environ.get("DATABASE_USERNAME", ""),
+        "PASSWORD": os.environ.get("DATABASE_PASSWORD", ""),
+        "HOST": os.environ.get("RDS_INSTANCE_ADDRESS", ""),
+        "PORT": "5432",
```

Comment:
Can this be made `1234` so aligns with the readme and works for local development without need for changing

Reply:
Not really. The value only needs to be changed when using a loopback connection and I hope the guidance is now a little clearer. I avoided creating another environment variable for the port as it will always be 5432 unless you need to target rds from your local development machine.